spec/lang/well-formed.md

```diff
@@ -51,7 +55,7 @@ impl SizeStrategy {
 impl PointeeInfo {
     fn check_wf<T: Target>(self) -> Result<()> {
         // We do *not* require that size is a multiple of align!
-        ensure_wf(matches!(self.size, SizeStrategy::Sized(_)), "PointeeInfo: Unimplemented size strategy found")?;
+        // TODO(UnsizedTypes): Ensure any future alignment strategy matches the size strategy.
```

Comment:
FWIW I am thinking we might have a `SizeAndAlign` type to represent both strategies, since they are coupled. But that's a future discussion :)